jax/_src/numpy/util.py

```diff
@@ -117,6 +119,27 @@ def _parse_extra_params(extra_params: str) -> dict[str, str]:
   return {p.partition(' : ')[0].partition(', ')[0]: p for p in parameters}
 
 
+def _place_array(x: Array, device: xc.Device | Sharding | None = None, copy=None) -> Array:
+  """Helper utility for copying an array, or placing it on a device or sharding.
+
+  This utility uses `jax.device_put` for device placement.
+  """
+  out = x
+  if device is not None:
+    # TODO(micky774): Add check to avoid error if no actual device transfer is
+    # necessary
+    if copy is not None and not copy:
+      raise ValueError(
+        f"Specified {device=} which requires a copy, however copy=False. Set "
+        "copy=True or copy=None to perform the requested operation."
+      )
+    out = api.device_put(out, device)
+
+  # TODO(micky774): Avoid copy if data has already been copied via device
```

Comment:
This todo doesn't make sense? In this branch the device is None. Remove the todo?